src/map/attackround.cpp

```diff
@@ -358,12 +360,31 @@ void CAttackRound::CreateKickAttacks()
             m_kickAttackOccured = true;
         }
 
-        // TODO: Possible Lua function for the nitty gritty stuff below.
-
-        // Mantra set mod: Try an extra left kick attack.
+        // Tantra set mod: Try an extra left kick attack.
         if (m_kickAttackOccured && dsprand::GetRandomNumber(100) < m_attacker->getMod(Mod::EXTRA_KICK_ATTACK))
         {
             AddAttackSwing(PHYSICAL_ATTACK_TYPE::KICK, LEFTATTACK, 1);
         }
     }
-}
\ No newline at end of file
+}
+
+/************************************************************************
+*																		*
+*  Creates a Daken throw.												*
+*																		*
+************************************************************************/
+void CAttackRound::CreateDakenAttack()
+{
+    if (m_attacker->objtype == TYPE_PC)
+    {
+        // if ammo slot is a shuriken (throwing, consumable)
```

Comment:
Where is it checking for shuriken? Right now I'm pretty sure this throws things like Tiphia Sting. RIP Tiphia Sting when Sange gets added.

Reply:
next line